docs/web3.geth.rst

```diff
@@ -309,7 +309,7 @@ The following methods are available on the ``web3.geth.personal`` namespace.
 
     Unlocks the given ``account`` for ``duration`` seconds.
     If ``duration`` is ``None``, then the account will remain unlocked
-    for 300 seconds (which is current default by Geth v1.9.5);
+    for 300 seconds (which is current default by Geth v1.10.15);
```

Comment:
Is 300 seconds for sure the current default?

Reply:
I did specifically verify that it's still 300 - https://github.com/ethereum/go-ethereum/blob/master/internal/ethapi/api.go